src/pyrqlite/cursors.py

```diff
@@ -163,8 +163,10 @@ def execute(self, operation, parameters=None, queue=False, wait=False):
 
         command = self._get_sql_command(operation)
         if command in ('SELECT', 'PRAGMA'):
-            payload = self._request("GET",
-                                    "/db/query?" + _urlencode({'q': operation}))
+            path = "/db/query?"
+            if consistency in ("none", "weak", "strong"):
+                path = path + "level=" + consistency
+            payload = self._request("GET", path + _urlencode({'q': operation}))
```

Comment:
Or do: _urlencode({'q': operation, 'consistency': c})) and default `consistency` to `weak`.

Reply:
Ah, glaring miss on my part. I see how the query is built now, I'll fix this as well when I add tests.

Reply:
I disagree about setting `consistency` to `weak` by default here. I think it's probably more appropriate to not set a consistency level param at all if one is not expressly provided by the user. From the rqlite docs, it seems that the default is `weak` if one isn't provided. If that default were to change for any reason in the future in the server, then it would also need to change here.

With that said, I'm happy to commit to setting that default if that's your preference.

Reply:
Yeah, that's fine. I think your position is just as correct TBH.

Reply:
So I suggest we create the query params dict, and set `q`. And only if the client sets `consistency` explicitly should the code also add a `consistency` key, so no consistency is set unless explicitly set by the client. That's what I might do anyway. 

By the way, to your point that things can change in the database itself, that argues against even checking if any supplied consistency is one of `none`, `weak`, or `strong`. While I doubt it will happen, rqlite mihgt support a new level in the future, and this client library would need to change again. Instead you just take whatever string is set for consistency (if it's not None) and just set it.

Reply:
That's valid. We'll rely on rqlite's error messages here if the user provides an invalid consistency level.